UniversityLibrary.UnitTests/CorrespondenceTest.cs

```diff
@@ -0,0 +1,45 @@
+using University;
+
+namespace UniversityLibrary.UnitTests
+{
+
+    [TestFixture]
+    public class CorrespondenceTest
+    {
+        [Test]
+        public void ConstructorTest()
+        {
+            var corstudent = GetTestCorStudent();
+
+            Assert.That(corstudent.Job, Is.EqualTo("Кафе Рандеву"));
+            Assert.That(corstudent.Position, Is.EqualTo("Официант"));
+        }
+
+        [Test]
+        public void GetInfo_CorStudent()
+        {
+            var corstudent = GetTestCorStudent();
+            var lines = new[]
+            {
+                "Никита Смирнов УГИ-235109",
+                "Номер зачётной книжки: 2023006 Институт: Гуманитарный Напрвление обучения: Бакалавр",
```

Comment:
Тест не проходит.

> Expected string length 83 but was 84. Strings differ at index 58.
  Expected: "...2023006 Институт: Гуманитарный Напрвление обучения: Бакалавр"
  But was:  "...2023006 Институт: Гуманитарный Направление обучения: Бакалавр"

Очевидною опечатка. Почему сами не исправили?
```suggestion
                "Номер зачётной книжки: 2023006 Институт: Гуманитарный Направление обучения: Бакалавр",
```